modules/apps/27-interchain-accounts/types/errors.go

```diff
@@ -8,7 +8,7 @@ var (
 	ErrUnknownDataType             = sdkerrors.Register(ModuleName, 2, "unknown data type")
 	ErrAccountAlreadyExist         = sdkerrors.Register(ModuleName, 3, "account already exist")
 	ErrPortAlreadyBound            = sdkerrors.Register(ModuleName, 4, "port is already bound")
-	ErrUnsupportedChain            = sdkerrors.Register(ModuleName, 5, "unsupported chain")
+	ErrInvalidChannelFlow          = sdkerrors.Register(ModuleName, 5, "invalid message sent to channel end")
```

Comment:
`ErrUnsupportedChain` was unused, so I replaced with the new error type which I copied from ccv as reference